scripts/upgrade

```diff
@@ -161,32 +162,34 @@ ynh_exec_warn_less ynh_install_composer --phpversion="$phpversion" --workdir="$f
 #=================================================
 # MODIFY A CONFIG FILE
 #=================================================
-ynh_script_progression --message="Modifying a config file..."
+#ynh_script_progression --message="Modifying a config file..."
 
-config="$final_path/.env"
-ynh_backup_if_checksum_is_different --file="$config"
-cp ../conf/.env "$config"
+#config="$final_path/.env"
+#ynh_backup_if_checksum_is_different --file="$config"
+#cp ../conf/.env "$config"
 
-ynh_replace_string --match_string="random_key" --replace_string="$random_key" --target_file="$config"
-ynh_replace_string --match_string="yunouser" --replace_string="$db_name" --target_file="$config"
-ynh_replace_string --match_string="yunopass" --replace_string="$db_pwd" --target_file="$config"
-ynh_replace_string --match_string="yunobase" --replace_string="$db_name" --target_file="$config"
-ynh_replace_string --match_string="yunomail" --replace_string="$email" --target_file="$config"
-ynh_replace_string --match_string="yunodomain" --replace_string="$domain" --target_file="$config"
-ynh_replace_string --match_string="yunoappurl" --replace_string="$domain$path_url" --target_file="$config"
+#ynh_replace_string --match_string="random_key" --replace_string="$random_key" --target_file="$config"
+#ynh_replace_string --match_string="yunouser" --replace_string="$db_name" --target_file="$config"
+#ynh_replace_string --match_string="yunopass" --replace_string="$db_pwd" --target_file="$config"
+#ynh_replace_string --match_string="yunobase" --replace_string="$db_name" --target_file="$config"
+#ynh_replace_string --match_string="yunomail" --replace_string="$email" --target_file="$config"
+#ynh_replace_string --match_string="yunodomain" --replace_string="$domain" --target_file="$config"
+#ynh_replace_string --match_string="yunoappurl" --replace_string="$domain$path_url" --target_file="$config"
 
 # Recalculate and store the checksum of the file for the next upgrade.
-ynh_store_file_checksum --file="$config"
```

Comment:
If you keep that part, in the future, upgrading .env will be more easy

Reply:
Restored the default upgrade checks and replacements.